packages/next/src/build/collect-build-traces.ts

```diff
@@ -298,15 +324,19 @@ export async function collectBuildTraces({
         '**/*.d.ts',
         '**/*.map',
         '**/next/dist/pages/**/*',
-        ...(ciEnvironment.hasNextSupport ? ['**/node_modules/sharp/**/*'] : []),
+        ...(ciEnvironment.hasNextSupport
+          ? ['**/node_modules/sharp/**/*', '**/@img/sharp-libvips*/**/*']
```

Comment:
Should this be `**/node_modules/@img/sharp-libvips*/**/*` instead (add the node_modules prefix)?

I'm also wondering why these need to be singled out. Couldn't we instead ignore `**/*.node`?

Reply:
since this name is more specific should be ok without `node_modules` but doesn't hurt to add it, we can't ignore all `.node` files unless in general I don't think as there are valid native modules when deployed

Reply:
In what case is a `.node` file "valid" vs "invalid?